ansible_runner/runner_config.py

```diff
@@ -189,7 +204,9 @@ def prepare(self):
 
         # Use local callback directory
         callback_dir = self.env.get('AWX_LIB_DIRECTORY', os.getenv('AWX_LIB_DIRECTORY'))
-        if callback_dir is None:
+        if callback_dir is None and self.containerized:
+            callback_dir = '/usr/lib/python3.6/site-packages/ansible_runner/callbacks'
```

Comment:
This assumes you've installed runner in your global `site-packages`, which might break some runner use cases (what about people who install runner into a virtualenv?)

Reply:
That gets messy because venv paths become part of the software interface. Job isolation will not be compatible with all images that have Ansible, we thought that EEs would have a fairly minimal set of requirements (ansible / runner), but I'm questioning that as we get into the details.

If you're running an image that uses venvs, then how does that get communicated from the image to `ansible-runner`? Even if the image itself had some metadata associated with it, runner would have to run docker/podman commands to find that out. See https://github.com/ansible/ansible-builder/issues/33 about venvs. 

This is a potential solution, take it with a _large_ grain of salt:

 - In the base image, copy the runner callback stuff to `/usr/share/ansible/plugins/callback`, since this is a default value of `ANSIBLE_CALLBACK_PLUGINS`
 - if the job is containerized, don't set this path at all, because there's no need to.

Considering the import hacks in the callback, I anticipate that this may take some work to get right. On the other hand... if we put the callback into a collection inside of `/usr/share/collections` this would all start to make a lot more sense.